BGAnimations/ScreenEvaluation common/Shared/RpgOverlay.lua

```diff
@@ -322,13 +476,15 @@ for player in ivalues(PlayerNumber) do
 		end,
 		-- White border
 		Def.Quad {
+			Name="MainBorder",
 			InitCommand=function(self)
 				self:diffuse(RpgYellow):zoomto(paneWidth + borderWidth, paneHeight + borderWidth + 1)
```

Comment:
Should the diffuse calls stay for those actors? Isn't Set*Style taking care of this?

Reply:
Removed